{"name": "Попытка входа с невалидными данными: случайные логин и пароль", "status": "passed", "steps": [{"name": "Подготовка тестовых данных для сценария: случайные логин и пароль", "status": "passed", "start": 1758210127995, "stop": 1758210127995}, {"name": "Отправляем POST-запрос для авторизации курьера", "status": "passed", "start": 1758210127995, "stop": 1758210128556}, {"name": "Проверяем код ответа 404", "status": "passed", "start": 1758210128556, "stop": 1758210128556}, {"name": "Проверяем сообщение об ошибке", "status": "passed", "start": 1758210128556, "stop": 1758210128556}], "parameters": [{"name": "test_case", "value": "'случайные логин и пароль'"}, {"name": "credentials", "value": "<function TestCourierLogin.<lambda> at 0x0000025C0D73CE00>"}, {"name": "expected_status", "value": "404"}, {"name": "expected_message", "value": "'Учетная запись не найдена'"}], "start": 1758210127995, "stop": 1758210128557, "uuid": "8e2e8d47-872e-4e57-86a6-dc172e1be4ca", "historyId": "f4a238b68989f8c05dc98a6a6c2585fb", "testCaseId": "505bba1c9113ea857d47e178537c8295", "fullName": "tests.test_courier_login.TestCourierLogin#test_login_negative_cases", "labels": [{"name": "story", "value": "Неуспешная авторизация"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_courier_login"}, {"name": "subSuite", "value": "TestCourierLogin"}, {"name": "host", "value": "What-Is-Lowe"}, {"name": "thread", "value": "5532-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_courier_login"}], "titlePath": ["tests", "test_courier_login.py", "TestCourierLogin"]}